sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -1797,11 +1810,11 @@ SIMPLE_COMMENT
     ;
 
 BRACKETED_EMPTY_COMMENT
```

Comment:
do we still need it?

Reply:
`(BRACKETED_COMMENT|.)*?` should work for empty comment

Reply:
No need.